avocado_vt/plugins/vt_resolver.py

```diff
@@ -45,6 +45,8 @@ def _get_reference_resolution(self, reference):
             if self.config.get("nrunner.max_parallel_tasks", 1) != 1:
                 warnings.warn("The VT NextRunner can be run only with "
                               "nrunner-max-parallel-tasks set to 1")
+                return ReferenceResolution(reference,
+                                           ReferenceResolutionResult.NOTFOUND)
```

Comment:
I'm wondering if "NOTFOUND" or "ERROR" is better here.

Reply:
I don't want to put "ERROR" here, because I can imagine a use-case when user have installed avocado-vt, but he is running non vt tests in parallel mode. IMO, in such case, the "ERROR" doesn't make sense.